app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -0,0 +1,185 @@
+  attribute :field_66, :integer
+  attribute :field_67, :integer
+  attribute :field_68, :integer
+  attribute :field_69, :integer
+  attribute :field_70, :integer
+  attribute :field_71, :integer
+  attribute :field_72, :integer
+  attribute :field_73, :integer
+  attribute :field_74, :integer
+  attribute :field_75, :integer
+  attribute :field_76, :integer
+  attribute :field_77, :integer
+  attribute :field_78, :integer
+  attribute :field_79, :integer
+  attribute :field_80, :decimal
+  attribute :field_81, :decimal
+  attribute :field_82, :decimal
+  attribute :field_83, :decimal
+  attribute :field_84, :decimal
+  attribute :field_85, :decimal
+  attribute :field_86, :integer
+  attribute :field_87, :integer
+  attribute :field_88, :decimal
+  attribute :field_89, :integer
+  attribute :field_90, :integer
+  attribute :field_91, :integer
+  attribute :field_92, :integer
+  attribute :field_93, :integer
+  attribute :field_94, :integer
+  attribute :field_95
+  attribute :field_96, :integer
+  attribute :field_97, :integer
+  attribute :field_98, :integer
+  attribute :field_99, :integer
+  attribute :field_100, :string
+  attribute :field_101, :integer
+  attribute :field_102, :integer
+  attribute :field_103, :integer
+  attribute :field_104, :integer
+  attribute :field_105, :integer
+  attribute :field_106, :integer
+  attribute :field_107, :string
+  attribute :field_108, :string
+  attribute :field_109, :string
+  attribute :field_110
+  attribute :field_111, :integer
+  attribute :field_112, :string
+  attribute :field_113, :integer
+  attribute :field_114, :integer
+  attribute :field_115
+  attribute :field_116, :integer
+  attribute :field_117, :integer
+  attribute :field_118, :integer
+  attribute :field_119, :integer
+  attribute :field_120, :integer
+  attribute :field_121, :integer
+  attribute :field_122, :integer
+  attribute :field_123, :integer
+  attribute :field_124, :integer
+  attribute :field_125, :integer
+  attribute :field_126, :integer
+  attribute :field_127, :integer
+  attribute :field_128, :integer
+  attribute :field_129, :integer
+  attribute :field_130, :integer
+  attribute :field_131, :string
+  attribute :field_132, :integer
+  attribute :field_133, :integer
+  attribute :field_134, :integer
+
+  validates :field_1, presence: true, numericality: { in: (1..12) }
+  validates :field_4, numericality: { in: (1..999), allow_blank: true }
+  validates :field_4, presence: true, if: :field_4_presence_check
+
+  validate :validate_possible_answers
+
+# delegate :valid?, to: :native_object
+# delegate :errors, to: :native_object
+
+private
+
+  def native_object
+    @native_object ||= LettingsLog.new(attributes_for_log)
+  end
+
+  def field_mapping
+    {
+      field_134: :renewal,
+    }
+  end
+
+  def validate_possible_answers
+    field_mapping.each do |field, attribute|
+      possible_answers = FormHandler.instance.current_lettings_form.questions.find { |q| q.id == attribute.to_s }.answer_options.keys
```

Comment:
Should we be getting the form based on the `year` in `bulk_upload` table?

Reply:
Unless it's out of scope, in which case feel free to ignore

Reply:
would it make sense to map through the native object and get the form from there?

Reply:
this code will be removed in favour of running validations on the `Log` and then proxying them thru to the relevant fields for the CSV upload. although those changes are outside the scope of this PR as the main focus of this PR is to simply get one error which can be surfaced in the UI